Sources/Defaults/Defaults.swift

```diff
@@ -63,6 +83,13 @@ public enum Defaults {
 			key.suite[key] = newValue
 		}
 	}
+
+	public static subscript<Value: Serializable & Codable>(key: Key<Value>) -> Value {
+		get { key.suite[key] }
+		set {
+			key.suite[key] = newValue
+		}
+	}
```

Comment:
What's the reason for this overload? The logic is the same, so I don't really see what it adds.